src/cpp/src/visual_language/inputs_embedder.cpp

```diff
@@ -291,6 +294,10 @@ class InputsEmbedderMiniCPM : public InputsEmbedder::IInputsEmbedder {
             m_image_id = 0;
         }
```

Comment:
I see `resample(...)` in the loop, which is embedding part IMO 

@Wovchena how do you think? I expect that tokenization should imply merging final embeddings only

Reply:
Yeah, but why is merging should be part of tokenization? I think the measurement should happen in `get_encoded_input_ids()`

Reply:
Not really. I was thinking about ignoring that stage :)

Reply:
what is the goal to break down 1st token at all and expose it partially (e.g. only image embedding), but not the whole breakdown?

As a user, I would expect either:
- sum up all stages in 1st token and it will result in 1 token latency
- hide all details about 1st token at all, as they are useless (?)

Reply:
the whole breakdown was considered as too detailed not needed report. But I don't know why we need VLM unique stage as a separate metric

Reply:
agreed on meeting, that we need to measure whole prompt embedding preparation stage which includes text tokenization + embedding, image embeddings and final merging.